### PR TITLE
The last merge conflict

### PR DESCRIPTION
**Pull request by sam0x17**
The Ballad of Unified Branches

devnet-ready and devnet clashed,
testnet’s trials and main’s steadfast.
A pull request, both bold and true,
Resolved their rift, their strength renewed.

No diff remains, no merge astray,
Peace crowned their code this final day.
devnet-ready, devnet, testnet, main—
Together now, in Git’s domain.

### CODE REVIEW

File: node/src/service.rs [hunk 1]
```diff
@@ -53,41 +49,41 @@ type GrandpaBlockImport<B, C> =
...
-    B: BlockT<Hash = H256>,
-    RA: ConstructRuntimeApi<B, FullClient<B, RA, HF>>,
-    RA: Send + Sync + 'static,
-    RA::RuntimeApi: RuntimeApiCollection<B, AuraId, AccountId, Nonce, Balance>,
-    HF: HostFunctionsT + 'static,
+    // B: BlockT<Hash = H256>,
+    // RA: ConstructRuntimeApi<Block, Client>,
+    // RA: Send + Sync + 'static,
+    // RA::RuntimeApi: RuntimeApiCollection<B, AuraId, AccountId, Nonce, Balance>,
+    // HF: HostFunctionsT + 'static,
```
Comment (orriin) [inline node/src/service.rs:78]: ?

File: node/src/service.rs [hunk 2]
```diff
@@ -191,21 +187,21 @@ where
...
-    RA: ConstructRuntimeApi<B, FullClient<B, RA, HF>>,
-    RA: Send + Sync + 'static,
-    RA::RuntimeApi: RuntimeApiCollection<B, AuraId, AccountId, Nonce, Balance>,
-    HF: HostFunctionsT + 'static,
+    // B: BlockT,
+    NumberFor<Block>: BlockNumberOps,
+    // RA: ConstructRuntimeApi<B, FullClient<B, RA, HF>>,
+    // RA: Send + Sync + 'static,
+    // RA::RuntimeApi: RuntimeApiCollection<B, AuraId, AccountId, Nonce, Balance>,
+    // HF: HostFunctionsT + 'static,
```
Comment (orriin) [inline node/src/service.rs:204]: 👀

File: node/src/service.rs [hunk 3]
```diff
@@ -242,20 +238,20 @@ where
...
-    RA::RuntimeApi: RuntimeApiCollection<B, AuraId, AccountId, Nonce, Balance>,
-    HF: HostFunctionsT + 'static,
+    _grandpa_block_import: GrandpaBlockImport<Block, Client>,
+) -> Result<(BasicQueue<Block>, BoxBlockImport<Block>), ServiceError>
+// where
+    // B: BlockT,
+    // RA: ConstructRuntimeApi<B, FullClient<B, RA, HF>>,
+    // RA: Send + Sync + 'static,
+    // RA::RuntimeApi: RuntimeApiCollection<B, AuraId, AccountId, Nonce, Balance>,
+    // HF: HostFunctionsT + 'static,
```
Comment (orriin) [inline node/src/service.rs:254]: 👀

File: node/src/service.rs [hunk 4]
```diff
@@ -269,25 +265,25 @@ where
...
-    RA::RuntimeApi: RuntimeApiCollection<B, AuraId, AccountId, Nonce, Balance>,
-    HF: HostFunctionsT + 'static,
-    NB: sc_network::NetworkBackend<B, <B as sp_runtime::traits::Block>::Hash>,
+    // B: BlockT<Hash = H256>,
+    NumberFor<Block>: BlockNumberOps,
+    // <B as BlockT>::Header: Unpin,
+    // RA: ConstructRuntimeApi<B, FullClient<B, RA, HF>>,
+    // RA: Send + Sync + 'static,
+    // RA::RuntimeApi: RuntimeApiCollection<B, AuraId, AccountId, Nonce, Balance>,
+    // HF: HostFunctionsT + 'static,
```
Comment (orriin) [inline node/src/service.rs:280]: 👀

**Review by gztensor — APPROVED** [review]

File: node/src/service.rs [hunk 5]
```diff
@@ -676,35 +665,31 @@ pub fn new_chain_ops(
...
     >,
 ) -> Result<(), ServiceError>
-where
-    B: BlockT,
-    RA: ConstructRuntimeApi<B, FullClient<B, RA, HF>>,
-    RA: Send + Sync + 'static,
-    RA::RuntimeApi: RuntimeApiCollection<B, AuraId, AccountId, Nonce, Balance>,
-    HF: HostFunctionsT + 'static,
+// where
+    // B: BlockT,
```
Comment (orriin) [inline node/src/service.rs:688]: 👀

File: node/src/service.rs (hunk 1 above)
Comment (ales-otf) [reply node/src/service.rs:78]: This is from a yesterday's quickfix. I'm going to clean everything, when it's merged

**Pull request merged**